apps/native/app/src/screens/passkey/passkey.tsx

```diff
@@ -79,8 +96,9 @@ export const PasskeyScreen: NavigationFunctionComponent<{
   const intl = useIntl()
   const theme = useTheme()
   const { openBrowser } = useBrowser()
+  const { height } = useWindowDimensions()
+  const isSmallDevice = height < 800
```

Comment:
If I need to use this in some other place I might consider moving this to a util and maybe implement a few breakpoints but will keep as is for now since this is the only place it is needed for now.